### PR TITLE
* Add structure for SMSG_COVENANT_PREVIEW_OPEN_NPC

### PR DESCRIPTION
**Pull request by Mistixx**
added structure for covenant open npc a sample

imagen_2021-05-11_040424

### CODE REVIEW

File: WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs [hunk 1]
```diff
@@ -361,6 +361,14 @@ public static void HandleGarrisonCovenantRenownSendCatchupState(Packet packet)
             packet.ResetBitReader();
             packet.ReadBit("CatchupState");
         }
+        
+        [Parser(Opcode.SMSG_COVENANT_PREVIEW_OPEN_NPC)]
+        public static void HandleGarrisonCovenantPreviewOpenNpc(Packet packet)
+        {
+            packet.ReadPackedGuid128("GUID");
+            packet.ReadUInt32("Index");
```
Comment (MaxtorCoder) [inline WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs:369]:
```suggestion
            packet.ReadInt32("CovenantId");
```

**Review by MaxtorCoder — CHANGES_REQUESTED** [review]

File: WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs (hunk 1 above)
Comment (Mistixx) [reply WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs]: thank yes I couldn't find a name for it.

Comment (MaxtorCoder) [reply WowPacketParserModule.V9_0_1_36216/Parsers/GarrisonHandler.cs]: Next time look in the handler. :)

**Comment by Mistixx** [comment]
or if it was my mistake I had it right in my structure I put it wrong when uploading the commit
hehehe it was my mistake I have it right but I made a mistake in putting UInt32

_int64 __fastcall sub_14097CAB0(__int64 a1, __int64 a2, __int64 a3, __int64 a4, int a5)
{
   *(_DWORD *)(a1 + 24) = a5;
   *(_QWORD *)(a1 + 8) = a2;
   *(_QWORD *)(a1 + 16) = a3;
   *(_QWORD *)a1 = off_142D91EE8;
   *(_DWORD *)(a1 + 48) = 0;
   *(_OWORD *)(a1 + 32) = 0ui64;
   CDataStore::ReadPackedGuid128(a4, a1 + 32);
   a5 = 0;
   CDataStore::ReadInt32(a4, &a5);
   *(_DWORD *)(a1 + 48) = a5;
   return a1;
}

**Pull request merged**